Mergeable/TwigMergeableObjectHandler.php

```diff
@@ -31,27 +26,21 @@ public function __construct(Container $container, array $configuration)
     {
         parent::__construct($container, $configuration);
 
+        $this->twig = $container->get('twig');
```

Comment:
Don't use the Symfony container but pass directly the `Twig\Environment` service directly.

Reply:
Can you change it to the abstract class too please ? 
Is it a big impact ? :thinking:

Reply:
This bundle works in sf3.4 ! Why this PR ? Fork the project.